### ISSUE TITLE
.si domains (Slovenia) expiration date is not parsed

### ISSUE THREAD

**Issue by a-stankevich** [shlin168/go-whois#2]
`.si` domains expiration date is not parsed

Example whois infomation
```
domain:         gov.si
registrar:      Ministrstvo za javno upravo - DI
registrar-url:  http://www.mju.gov.si/
nameserver:     ns1s.gov.si (163.159.0.120)
nameserver:     ns2s.gov.si (163.159.0.121)
nameserver:     ns3s.gov.si (163.159.64.120)
ds:             44047 8 1 8BFA0A71784E2D3AD39C015BC8FFACD749951FFB
ds:             44047 8 2 6A93E1571F7E4F8C2C30BEF39E9935D76A80A1C9685B834F75EFDA69214FE822
registrant:     G31117
status:         ok
created:        1998-04-08
expire:         2023-06-06
source:         ARNES
```

go-whois result
```
> whois -q gov.si
time="2022-11-07T17:08:37+01:00" level=info msg="perform WHOIS query" public_suffixs="[gov.si]" query=gov.si
time="2022-11-07T17:08:38+01:00" level=info msg=parse parser=default tld=si
{
"parsed": {
"domain": "gov.si",
"registrar": {
"name": "Ministrstvo za javno upravo - DI"
},
"name_servers": [
"ns1s.gov.si (163.159.0.120)",
"ns2s.gov.si (163.159.0.121)",
"ns3s.gov.si (163.159.64.120)"
],
"created_date": "1998-04-08T00:00:00+00:00",
"statuses": [
"ok"
]
},
"whois_server": "whois.arnes.si",
"rawtext": "%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n% This is ARNES whois database\n%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n% Rights restricted by copyright.\n%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n% The WHOIS service offered by Arnes, .si Registry, is\n% provided for information purposes only. It allows persons\n% to check whether a specific domain name is still available\n% or not and to obtain information related to the registration\n% records of existing domain names.\n%\n% This WHOIS service accepts and displays only ASCII characters.\n\n% By submitting a query to our WHOIS lookup service, you agree\n% to the restrictions on the use of WHOIS and WHOIS data as follows:\n%      o You may not allow, enable or otherwise support\n%        the transmission of unsolicited, commercial advertising\n%        or other solicitations, whether via email or otherwise;\n%      o You may not use the information to target advertising\n%        in any possible way;\n%      o You may not cause inconvenience the domain name holders\n%        in any possible way by sending them messages\n%        (whether by automated, electronic processes capable of\n%        enabling high volumes or other possible means);\n%      o You may not copy, extract and/or publish contents\n%        of the WHOIS database.\n%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n\ndomain:\t\tgov.si\nregistrar:\tMinistrstvo za javno upravo - DI\nregistrar-url:\thttp://www.mju.gov.si/\nnameserver:\tns1s.gov.si (163.159.0.120)\nnameserver:\tns2s.gov.si (163.159.0.121)\nnameserver:\tns3s.gov.si (163.159.64.120)\nds:\t\t44047 8 1 8BFA0A71784E2D3AD39C015BC8FFACD749951FFB\nds:\t\t44047 8 2 6A93E1571F7E4F8C2C30BEF39E9935D76A80A1C9685B834F75EFDA69214FE822\nregistrant:\tG31117\nstatus:\t\tok\ncreated:\t1998-04-08\nexpire:\t\t2023-06-06\nsource:\t\tARNES\n\nDomain holder:\nNOT DISCLOSED\n\nTech:\nNOT DISCLOSED\n\n%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n% Full domain details are available here http://www.registry.si\n%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n",
"available": false
}
```

**Comment by shlin168** [comment]
Fix in c78f42449eaf2e539ab231795f95f9bbdbfd0079, thanks for the notification.
Also modify default read timeout since whois server for `.si`(`whois.arnes.si`) frequently timeout in my timezone...

**Comment by a-stankevich** [comment]
It works perfectly now, thank you!
